### DISCUSSION TITLE
`make fmt` failed due to out-of-date version of cue binary

### DISCUSSION THREAD

**Discussion by zzxwill** [kubevela/kubevela#2287] [Q&A] [answered]
**Describe the bug**

`make fmt` failed.

```
➜  kubevela git:(terraform) make fmt
go fmt ./...
/usr/local/bin/goimports -local github.com/oam-dev/kubevela -w $(go list -f {{.Dir}} ./...)
/usr/local/bin/cue fmt ./vela-templates/definitions/internal/
build constraints exclude all CUE files in ./vela-templates/definitions/internal (ignored: vela-templates/definitions/internal/annotations.cue, vela-templates/definitions/internal/apply-application.cue, vela-templates/definitions/internal/apply-component.cue, vela-templates/definitions/internal/apply-remaining.cue, ...)
make: *** [fmt] Error 1
```

**To Reproduce**

1. Clone this repo
2. run `make fmt`

**Expected behavior**

**Screenshots**

**KubeVela Version**

**Cluster information**

**Additional context**

The root cause is the version, for example `0.2.2` of cue is out-of-date. Upgrade it to `0.3.0` can fix this issue.

**Comment by wonderflow** [comment, marked as answer]
The root cause is the version, for example 0.2.2 of cue is out-of-date. Upgrade it to 0.3.0 can fix this issue.
